files/build_templates/docker_image_ctl.j2

```diff
@@ -89,6 +89,19 @@ function preStartAction()
     updateSyslogConf
 }
 
+{%- if docker_container_name == "database" %}
+
+function setPlatformLagIdBoundaries()
+{
+    CHASSIS_CONF=/usr/share/sonic/device/$PLATFORM/chassisdb.conf
+    if [ -f "$CHASSIS_CONF" ]; then
+        source $CHASSIS_CONF
+        $SONIC_DB_CLI CHASSIS_APP_DB SET "SYSTEM_LAG_ID_START" "$lag_id_start"
```

Comment:
Could you add these parameters viz lag_id_start, lag_id_end as needed in the chassisdb.conf in the VOQ LAG HLD if it is not present ?

Reply:
Yes. Updated section 3.3 System LAG ID management section of "Distributed VOQ HLD" Azure/SONiC#697  to mention the file chassisdb.conf and these parameters.

Reply:
Looks ok thanks ! Also what happens in case the user didn't specify the START and END limits in chassis_db.conf file. Should we define a default set of values here, just before we source the chassis conf ? OR we expect the user to always specify it in chassis db for it to work correctly.

Reply:
If chassisdb.conf does not specify lag_id_start and lag_id_end, chassis db will not have these set. So lag id allocator will return error and port channel creation will not proceed further without lag id (aka system port aggregator id)  since system port aggregator id is a mandatory attribute expected by SAI in voq chassis systems. Therefore, with voq chassis system these two parameters are mandatory to be specified by the user based on platform capability. Since this is platform dependent, we'll not be able to choose a default unless all vendors agree on some default. Until now we have not thought about a default starting and ending lag id.